db/migrations/004_incidents.sql

```diff
@@ -137,13 +150,27 @@ CREATE TABLE evidences (
   id UUID DEFAULT generate_ulid() PRIMARY KEY,
   description TEXT NOT NULL,
   hypothesis_id UUID NOT NULL,
+  config_id UUID NULL,
+  config_change_id UUID null,
+  config_analysis_id UUID null,
+  component_id UUID null,
+  check_id UUID null,
+  definition_of_done boolean, -- This indicates this item as needing to be fixed before closing the incident
+  done boolean, -- The evidence is done / resolved
+  factor boolean,
+  mitigator boolean,
```

Comment:
Should we store these in a json format ? Is there a possibility that more similar fields would be added ?

Reply:
These are core fields used for incident lifecycle automation, and later for machine learning which means that would be queried alot